.github/workflows/ci.yml

```diff
@@ -179,12 +179,12 @@ jobs:
           # high-compression tar.zst + zip for release, fast tar.zst otherwise
           if [[ '${{ startsWith(github.ref, 'refs/tags/v') && matrix.release }}' == true || -n '${{ needs.set-nightly.outputs.nightly }}' ]]; then
             tar cf - $dir | zstd -T0 --no-progress -19 -o pack/$dir.tar.zst
-            zip -rq pack/$dir.zip $dir
           else
             tar cf - $dir | zstd -T0 --no-progress -o pack/$dir.tar.zst
           fi
+          zip -rq9 pack/$dir.zip $dir
       - uses: actions/upload-artifact@v2
-        if: matrix.release
+        if: matrix.release && github.event_name == 'push' && github.repository == 'leanprover/lean4'
```

Comment:
Why? This is completely harmless.

Reply:
the TPIL trigger triggers a build over there and we don't want TPIL builds picking up incomplete pull requests.

Reply:
I always thought of github `build-artifact` as something that belongs to the team members who own the CI workflow.  Just downloading a github `build-artifact` is no simple task from the `gh` command line.

Reply:
I see I didn't realize you could download them from here https://github.com/leanprover/lean4/actions/runs/2585458255, that's cool.   For me personally I've never used these because I always build my PR locally and test it before pushing a PR.  But I could see the cross-platform aspect of the binaries could be handy for folks that want to share PR bits with other team members who are on different platforms...  I'm happy to put that back if you think it's useful.

Reply:
PS: the other reason I'm playing with removing this is to see if it solves this kind of thing over in TPIL: https://github.com/leanprover/theorem_proving_in_lean4/actions/runs/2582579057

Reply:
@lovettchris I want to clarify this part

> Leo wanted faster than nightly feedback on breaks to TPIL,

Nightly feedback is fine. The original issue I had is that I was not getting any feedback when changes on Lean broke TPIL.

Reply:
Ok, with this information I can instead change TPIL to stop using these build-artifacts, and we can cancel this PR.  I will change the TPIL workflow to grab the zip files from https://github.com/leanprover/lean4-nightly/releases

Reply:
Yes, the nightly builds are good enough. I will close this PR.